DetDescr/src/DetDescr/EcalGeometry.cxx

```diff
@@ -576,29 +533,29 @@ void EcalGeometry::buildNeighborMaps() {
                                         // corner of center module
     double specialY = moduler_ - 0.5 * cellR_;
     EcalID specialCellModuleID = getCellModuleID(specialX, specialY);
-    std::cout << "The neighbors of the bin in the upper-right corner of the "
-                 "center module, with cellModuleID "
-              << specialCellModuleID << " include " << std::endl;
+    ldmx_log(debug) << "The neighbors of the bin in the upper-right corner of
+  the " "center module, with cellModuleID "
+              << specialCellModuleID << " include " ;
     for (auto centerNN : NNMap_.at(specialCellModuleID)) {
-      std::cout << " NN " << centerNN
-                << TString::Format(" (x,y) (%.2f, %.2f)",
+      ldmx_log(debug) << " NN " << centerNN
+                << (" (x,y) (%.2f, %.2f)",
                                    getCellCenterAbsolute(centerNN).first,
                                    getCellCenterAbsolute(centerNN).second)
-                << std::endl;
+                ;
     }
     for (auto centerNNN : NNNMap_.at(specialCellModuleID)) {
-      std::cout << " NNN " << centerNNN
-                << TString::Format(" (x,y) (%.2f, %.2f)",
+      ldmx_log(debug) << " NNN " << centerNNN
+                << (" (x,y) (%.2f, %.2f)",
                                    getCellCenterAbsolute(centerNNN).first,
                                    getCellCenterAbsolute(centerNNN).second)
-                << std::endl;
+                ;
     }
-    std::cout << TString::Format(
+    ldmx_log(debug) << (
                      "This bin is a distance of %.2f away from a module edge. "
                      "Decision isEdge %d.",
                      distanceToEdge(specialCellModuleID),
                      isEdgeCell(specialCellModuleID))
-              << std::endl;
+              ;
```

Comment:
It looks like these changes are just caught-up in some find+replace. Not sure if we want to keep this commented-out code that just writes out all the tables, maybe we move it to a Print function that isn't called anywhere but could be called by a debugger later?